README.md

```diff
@@ -847,6 +891,42 @@ Controls whether the build will continue if lint identifies code that violate ch
 
 Default: `true`
 
+#### workingDirectoryRelativeToBasedir ####
+
+Controls the working directory relative to the `${project.basedir}` when examining monorepo dependency modules for copying source files with the `containerize-dependencies` goal. The working directory should be the `${project.basedir}`, hence the `null` default.
+
+Note: this parameter should be consistent with the `workingDirectory` configuration, such that both parameters point to the same expected working directory. For example, if the `workingDirectory` was overriden to `${project.basedir}/custom-workdir`, then `workingDirectoryRelativeToBasedir` should be set to `/custom-workdir`. Setting this parameter should only be necessary if a non-default `workingDirectory` configuration was set.
+
+Default: `null`
+
+#### distDirectoryRelativeToBasedir ####
+
+Controls the expected dist directory relative to the `${project.basedir}` when examining monorepo dependency modules for copying source files with the `containerize-dependencies` goal.
+
+Note: this parameter should be consistent with the `distDirectory` configuration. For example, if the `distDirectory` was set to `${project.basedir}/custom-dist`, then `distDirectoryRelativeToBasedir` should be set to `/custom-dist`. Setting this parameter should only be necessary if a non-default `distDirectory` configuration was set.
+
+Default: `/dist`
+
+#### targetDirectoryRelativeToBasedir ####
+
+Controls the expected target directory relative to the `${project.build.directory}` when examining monorepo dependency modules for copying source files with the `containerize-dependencies` goal.
+
+Note: this parameter should be consistent with the `targetDirectory` configuration. For example, if the `targetDirectory` was overriden to `${project.basedir}/custom-target`, then `targetDirectoryRelativeToBasedir` should be set to `/custom-target`. Setting this parameter should only be necessary if a non-default `targetDirectory` configuration was set.
+
+Default: `/target`
+
+#### anchorSourceDirectory ####
+
+Controls the directory of where containerization files will be copied from with the `containerize-dependencies` goal. If set, this value should be a directory path that houses all the necessary monorepo dependency Habushu modules (including transitive monorepo dependency modules). Additionally, if set, this path should either be an absolute path or a path relative to the `${project.basedir}` of the `containerize-dependencies` goal's execution. Typically, it is not recommended to set this parameter, as the default directory will be sufficient for monorepo use-cases.
+
+Default: the build's execution root directory
+
+#### containerizeSupportDirectory ####
+
+Controls the location of where containerization files will be placed to with the `containerize-dependencies` goal.
```

Comment:
will be placed to with the `containerize-dependencies` goal. -> will be copied to as part of the `containerize-dependencies` goal.